### PR TITLE
support standard "cssFloat" css property

### PR DESCRIPTION
**Pull request by stevenvachon**
For tools that compile CSS may have already converted the name.

More info:
https://npmjs.com/camelcase-css
https://npmjs.com/postcss-js

### CODE REVIEW

File: src/renderers/dom/shared/CSSPropertyOperations.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ var CSSPropertyOperations = {
         styles[styleName],
         component
       );
-      if (styleName === 'float') {
+      if (styleName === 'float' || styleName === 'cssFloat') {
         styleName = styleFloatAccessor;
       }
```
Comment (zpao) [inline src/renderers/dom/shared/CSSPropertyOperations.js]: We only need to set styleName to cssFloat if it isn't already cssFloat, so this doesn't actually do anything. i appreciate the thought though.

Comment (stevenvachon) [inline src/renderers/dom/shared/CSSPropertyOperations.js]: It sets it to `styleFloat` in IE8.

**Comment by zpao** [comment]
Ah true, that's what I get for trying to review from my phone 😬

**Comment by stevenvachon** [comment]
:) np

**Comment by zpao** [comment]
Thanks!

**Pull request merged**